release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -3415,11 +3415,13 @@ $ oc adm release info 4.15.18 --pullspecs
 The following enhancements are included in this z-stream release:
 
 [id="ocp-4-15-18-unservable-future-versions-status-condition"]
-===== Introducing the UnservableFutureVersions status condition for SHA1 routes
+===== Blocking upgrades for SHA1 serving certificates on routes and Ingress controllers
 
-* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to {product-version} causes routes with SHA1 certificates to be rejected.
+* {product-title} {product-version} does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to 4.16 cause routes and Ingress controllers with SHA1 certificates to be rejected.
```

Comment:
The intention is to indicate that the next version (4.16) of OpenShift doesn't support SHA1 certs. This version, 4.15 does. I believe `{product-version}` substitutes to 4.15. Is there a way to do `{product-version+1}`?

```suggestion
* {product-title} 4.16 does not support SHA1 certificates on routes. Consequently, upgrades from 4.15 to 4.16 cause routes and Ingress controllers with SHA1 certificates to be rejected.
```